frontend-react/src/hooks/UseValueSets.test.ts

```diff
@@ -101,28 +64,10 @@ describe("useValueSetsMeta", () => {
     });
 
     test("returns empty metadata when the passed table name doesn't exist in returned list of tables", async () => {
-        const { result, waitFor } = renderWithQueryWrapper();
+        const { result, waitFor } = renderWithQueryWrapper("unknown_table");
         await waitFor(() => !!result.current.valueSetMeta);
         expect(result.current.valueSetMeta).toEqual({});
     });
-
-    test("returns error when table list call errors", async () => {
```

Comment:
Yeah, I've been contemplating how to handle it since technically it'd have to be called in a component _then_ wrapped. I'm gonna dig to see if `renderHook` is something we can do in a try-catch, or maybe RTL has thought of this and have an ErrorBoundary-like catch system we can utilize.

Reply:
Should be one of the last things left to tackle on this PR